### ISSUE TITLE
[Feature]: Make Alt Text visible

### ISSUE THREAD

**Issue by trishanderton** [mastodon/mastodon-ios#852]
### Description

Alt Text should be visible without a screen reader so people can boost images that have it and avoid those that don't. 

### Suggested Solution

A Twitter-style Alt Text badge would be a great solution.

### Alternatives

MetaText shows Alt Text at the bottom if you click on the image, but this is less efficient because you have to click on every image to see whether it has it.

**Comment by j-f1** [comment]
Fixed by #699, which will be coming out in the next update!

**Comment by trishanderton** [comment]
> Fixed by #699, which will be coming out in the next update!

THIS IS FANTASTIC NEWS. Thank you!
